front/app/containers/Admin/projects/project/general/components/GeographicAreaInputs.tsx

```diff
@@ -14,11 +14,51 @@ import { IAreaData } from 'services/areas';
 import { useParams } from 'react-router-dom';
 import { TOnProjectAttributesDiffChangeFunction } from '..';
 
+// styles
+import styled from 'styled-components';
+import { fontSizes, colors } from 'utils/styleUtils';
+
 interface Props {
   areaIds: string[] | undefined;
   onProjectAttributesDiffChange: TOnProjectAttributesDiffChangeFunction;
 }
 
+const LabelText = styled.div`
+  display: flex;
+  flex-direction: column;
+  margin-top: -2px;
+
+  &.disabled {
+    opacity: 0.4;
+    cursor: not-allowed;
+  }
+
+  .header {
+    padding: 0;
+    margin: 0;
+    margin-bottom: 3px;
+    font-weight: 600;
+    font-size: ${fontSizes.base}px;
+  }
+
+  .description {
+    color: ${colors.adminSecondaryTextColor};
+  }
+`;
+
+const LabelHeaderDescription = ({
+  header,
+  description,
+}: {
+  header: JSX.Element;
+  description: JSX.Element;
+}) => (
+  <LabelText>
+    <span className="header">{header}</span>
+    <span className="description">{description}</span>
+  </LabelText>
+);
```

Comment:
I would import the one that we already have instead of duplicating it.

`import { LabelHeaderDescription } from './labels';
`